### PR TITLE
Update README docs

### PR DESCRIPTION
**Pull request by asteinhart**
## What does this PR do?
* update README.md to be more external facing
* Adds a CONTRIB.md file to add more internal notes

### CODE REVIEW

File: resources/d3.md
Comment (michplunkett) [inline resources/d3.md:1]: If the files are gonna be empty, I'd hesitate to add them until you have some copy to put in them.

File: CONTRIB.md [hunk 1]
```diff
@@ -1,33 +1,128 @@
...
+    - You can run `git status` to see which files in red have not been added. Then run `git add [file-name]` for each file to add them. Next run `git commit -m '[commit message]'` to commit your changes. Then run `git push` to push your changes. If this is your first commit, github may ask you to run a different command, which you should. Ex `git push --set-upstream origin [branch-name]`
+
+## The Chicago Maroon’s Data Contribution and Style Guide
+
+### Contribution Standards
+
+#### Pull Request Titles
 
 They should be written in the present imperative tense. Examples of that tense are:
 "Fix issue in the dispatcher where…", "Improve our handling of…", etc."
```
Comment (michplunkett) [inline CONTRIB.md:98]: PR titles are truncated at 70 characters. I'd add something about that in here as well as the PR template.

File: ADMIN.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## The Visuals are served via this URL template
+
+URL Template: `https://chicagomaroon.github.io/data-visualizations/[PATH TO FILE]`
+
+URL Example: `https://chicagomaroon.github.io/data-visualizations/2023/thifting-growth/thifting-growth.min.html`
+
+## Commands
+
+-   `npm run lint`: Runs the formatter and linter in the repository and will let you know if any JavaScript faux pas were made in your code
```
Comment (michplunkett) [inline ADMIN.md]: This will need to be updated when https://github.com/chicagomaroon/data-visualizations/pull/97 is merged.

File: ADMIN.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    <iframe
+        class="data-viz-container"
+        src="https://chicagomaroon.github.io/data-visualizations/[PATH TO MINIFIED HTML FILE]"
+    ></iframe>
+</div>
+<div class="data-viz-description-container">
+    <div class="data-viz-credit">
+        <a href="https://chicagomaroon.com/staff_name/[FIRST NAME]-[LAST NAME]/"
+            >[AUTHOR OF CHART]</a
+        >
```
Comment (michplunkett) [inline ADMIN.md:31]: The formatter here looks a bit strange.

File: CONTRIB.md [hunk 4]
```diff
@@ -1,33 +1,128 @@
...
+
+5. After you have cloned the repo and downloaded Node, go to the base folder of the `data-visualizations` repository and run the command `npm ci` to install all package dependencies. You will only need to do this once
+
+### Tools and Skills to Review
+
+Programming is a skill that you learn by doing. There is no expectation that someone fully understands a programming language or package before they can start creating things. Much of the process is about learning as you go and building a muscle of problem solving and debugging as you go.
+
+With that being said, use the resources below as a starting place and reference source. Aim to learn just enough to jump into a project and learn the rest as you go.
+
+-   We primarily use [`HighCharts`](https://www.highcharts.com/), a JavaScript library, to build visualizations. Each HighChart graphic is embedded on a simple HTML page that is linked in its parent story.
```
Comment (michplunkett) [inline CONTRIB.md:50]: 🔬  We don't line breaks between the main points and subpoints throughout the rest of the documentation. I'd prefer to keep it that way, but if we are going to add the line break then it should be consistent through the rest of the documentation as well.

Comment (asteinhart) [reply CONTRIB.md:50]: this doenst make a difference in the render of the md as far i can tell

File: CONTRIB.md (hunk 1 above)
Comment (asteinhart) [reply CONTRIB.md:98]: i dont think anyone will be counting 70 characters, if its too long, a v quick fix in github

File: resources/d3.md
Comment (asteinhart) [reply resources/d3.md:1]: will be added be end of summer

**Pull request merged**